python/ray/util/sgd/v2/utils.py

```diff
@@ -18,6 +19,36 @@ def get_address_and_port() -> Tuple[str, int]:
     return addr, port
 
 
+def construct_path(path: Path, parent_path: Path) -> Path:
+    """Constructs a path relative to a parent.
+
+    Args:
+        path: A relative or absolute path.
+        parent_path: A relative path or absolute path.
+
+    Returns: An absolute path.
+    """
+    if path.expanduser().is_absolute():
+        return path.expanduser().resolve()
+    else:
+        return parent_path.joinpath(path).expanduser().resolve()
+
+
+def construct_path_with_default(path: Optional[Path], parent_path: Path,
```

Comment:
Minor: is this function necessary? Can the logic here be done directly in the executor?

Reply:
Good point, this made more sense when there were multiple flows using this. Removed and cleaned up!